### PR TITLE
(test PR, do not merge)

### PR DESCRIPTION
**Pull request by shinenelson**
partial resolution for #125

image

It would be nice if we could use a _symbolic_ logo of npmhub before the anchor link in the repository sidebar. That could make it consistent with the other links above it.

I tried injecting the SVG logo from the repository, but it would not work ( as in not displayed ). Maybe I as doing something wrong with injecting SVGs via JavaScript.

Also, if we had the symbolic icon, we could inject it somewhere in the header of the file : 
image
Potentially on the right of the contributors row since that area is empty.

### CODE REVIEW

**Comment by fregante** [comment]
If you can make it work, add it in the Svelte files, not outside. If not, I can't fix it

**Comment by shinenelson** [comment]
Simply moving the code from one file to another worked; I do not know whether you were implying to modify the code to work differently though.

**Comment by fregante** [comment]
I mean it needs to be done the Svelte way, which looks like HTML as you can see. It probably belongs to Box.svelte

**Comment by fregante** [comment]
Ahhhh I'm sorry. I didn't understand what it was doing, only now I'm seeing the screenshot. 

Then yeah this needs to be its own Svelte component, which should not be hard:

- in the main js file, duplicate the injection code, but instead of injecting App.svelte at the bottom it should inject MetaLink.svelte in the sidebar
- in MetaLink.svelte you basically just include some HTML like you see done in App.svelte, it doesn't need to be dynamic but it has to follow the same format as in App.svelte

**Comment by fregante** [comment]
I think the contents of MetaLink.svelte will just literally be:

``` html
<a href="#dependencies">npmhub</a>
```

Maybe also include the SVG

**Comment by shinenelson** [comment]
screenshot : `Promise { <pending> }`

File: source/components/MetaLink.svelte [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<h3 class="sr-only">Dependency List via N P M Hub</h3>
+<div class="mt-3">
+  <a href="#npmhub" class="Link--muted">
+    <svg version="1.1" aria-hidden="true" height="16" width="16" viewBox="150 50 250 450" data-view-component="true" class="octicon mr-2">
```
Comment (fregante) [inline source/components/MetaLink.svelte:4]: What's that data view component attribute?

Comment (shinenelson) [reply source/components/MetaLink.svelte:4]: I duplicated the SVG attributes from the other octicons. I was thinking that GitHub might have some use for it like the `data-pjax-*` attribute. If it is irrelevant, I can remove it.

Comment (fregante) [reply source/components/MetaLink.svelte:4]: You can get the actual icons from https://primer.style/octicons/

Comment (shinenelson) [reply source/components/MetaLink.svelte:4]: octicons would not have the icon for npmhub :confused: 

What I meant was that I _copied_ the SVG **attributes** that were on the octicons ( for Readme and License ) and used them on the npmhub icon.

Did you not notice `Promise { <resolved> }` on the screenshot above? :stuck_out_tongue_closed_eyes:

PS : I know about primer.style. I have it open usually when I am contributing to Refined GitHub.

**Comment by fregante** [comment]
Sorry @shinenelson can you open a new PR with my latest changes (except App.svelte)?

I found bug in Refined GitHub and I'm going to keep this PR as testing ground.

File: source/components/MetaLink.svelte (hunk 1 above)
Comment (fregante) [reply source/components/MetaLink.svelte:4]: > Did you not notice `Promise { <resolved> }` on the screenshot above? 😝

well don't do that, no idea of what it means. Just send a comment when you have the screenshot

Comment (fregante) [reply source/components/MetaLink.svelte:4]: But I see it now. I like the icon 👍

**Pull request closed without merging**